k8s/network-policies/viz-network-policy.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: networking.k8s.io/v1
```

Comment:
this doesn't really do what we want right ? What we want is a CSP on the actual `/content` doc so the the client-side code can't access internet (not the viz service)